litmus-portal/frontend/src/utils/promUtils.ts

```diff
@@ -524,67 +529,49 @@ export const chaosEventDataParserForPrometheus = (
-              value: latestRunMetric
-                ? latestRunMetric.probeSuccessPercentage
-                : '--',
-            },
-            {
-              subDataName: 'Experiment Verdict',
-              value: latestRunMetric
-                ? latestRunMetric.experimentVerdict +
-                  (latestRunMetric.experimentVerdict ===
-                    CHAOS_EXPERIMENT_VERDICT_PASS ||
-                  latestRunMetric.experimentVerdict ===
-                    CHAOS_EXPERIMENT_VERDICT_FAIL
-                    ? 'ed'
-                    : '')
-                : '--',
-            },
-          ],
-          // Filter subData within the start and end time of interleaving on experiment's lastUpdatedTimeStamp.
-          // Add one extra subData field - lastUpdatedTimeStamp to filter subData in graph.
-          // This method sends the latest run details within selected dashboard time range as the subData.
-          // Needs to be updated to send every run detail with lastUpdatedTimeStamp in the ${NEW_SUBDATA_FIELD}.
-          /* 
-            Schema: 
-            (to be updated as)
-            - subData: [
-                { subDataName: "subData-1-1", value: "1-1", lastUpdatedTimeStamp: 1616832979 },
-                { subDataName: "subData-1-2", value: "1-2", lastUpdatedTimeStamp: 1616833006 },
-              ],
-            subData: 
-            (to be filtered by lastUpdatedTimeStamp in litmus-ui graph on hovering over chaos events)
-            - Workflow Status 
-            - Experiment Status
-            - Resilience Score
-            - Probe Success Percentage
-            - Experiment Verdict
-            */
+          subData: allRunMetric
+            .map((elem: RunWiseChaosMetrics) => {
+              return [
+                {
+                  subDataName: 'Workflow Status',
+                  value: elem ? elem.workflowStatus : STATUS_RUNNING,
+                  date: elem ? elem.lastUpdatedTimeStamp : 0,
+                },
+                {
+                  subDataName: 'Experiment Status',
+                  value: elem ? elem.experimentStatus : STATUS_RUNNING,
+                  date: elem ? elem.lastUpdatedTimeStamp : 0,
+                },
+                {
+                  subDataName: 'Resilience Score',
+                  value:
+                    elem &&
+                    elem.workflowStatus !== STATUS_RUNNING &&
+                    elem.resilienceScore !== INVALID_RESILIENCE_SCORE_STRING
+                      ? elem.resilienceScore
+                      : '--',
+                  date: elem ? elem.lastUpdatedTimeStamp : 0,
+                },
+                {
+                  subDataName: 'Probe Success Percentage',
+                  value: elem ? elem.probeSuccessPercentage : '--',
+                  date: elem ? elem.lastUpdatedTimeStamp : 0,
+                },
+                {
+                  subDataName: 'Experiment Verdict',
```

Comment:
add translation

Reply:
This is neither a react function nor a custom hook how to add a translation?

Reply:
The only way to add translation here it by adding translation path here and then while constructing the chaos_data we have to map again over every element and then add a translation hook for all sub-data

Reply:
subData is used in litmus-ui...there we don't have a translation file to process them @ishangupta-ds

Reply:
Have suggested a way to @whitetiger1399 cc: @arkajyotiMukherjee